Dockerfiles/Dockerfile

```diff
@@ -10,9 +10,8 @@ RUN export ARCH=$(case $(arch) in x86_64) echo -n amd64 ;; aarch64) echo -n arm6
     curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH};\
     chmod +x operator-sdk_${OS}_${ARCH};\
     mv operator-sdk_${OS}_${ARCH} operator-sdk;\
-    mv operator-sdk /usr/local/bin/;
```

Comment:
Use `&&` instead of `;` for chain of commands.

Reply:
export COMMAND does not work with && . Rest of the places i have updated required.

Reply:
because export is not a command :-) and it is fine to use it there